test/unit/org/apache/cassandra/service/StorageServiceTest.java

```diff
@@ -298,12 +298,21 @@ public void testBatchSizeWarnThresholdInKiB()
                 storageService.setBatchSizeWarnThresholdInKiB(2 * 1024 * 1024);
                 fail("Should have received an IllegalArgumentException batch_size_warn_threshold = 2GiB");
             }
-            catch (IllegalArgumentException ignored) { }
+            catch (IllegalArgumentException ignored)
+            {
+            }
             Assert.assertEquals(1024, storageService.getBatchSizeWarnThresholdInKiB());
         }
         finally
         {
             storageService.setBatchSizeWarnThresholdInKiB(previousBatchSizeWarnThreshold);
         }
     }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testLocalDatacenterNodesExcludedDuringRebuild()
+    {
+        StorageService service = StorageService.instance;
+        service.rebuild(DatabaseDescriptor.getLocalDataCenter(), "StorageServiceTest", null, null, true);
+    }
```

Comment:
Prefer to use 
```
    @Rule
    ExpectedException expectedException = ExpectedException.none();
```

This way, you can assert the exception type and the exception message too.

Reply:
Right, but the point is to assert the exception message too. `AssertJ` api can also do it.

Reply:
My vote is for `assertj` . I think we've started seeing more tests in the Cassandra codebase moving to `assertj` which is great!

Reply:
I believe we don't have assertThrows in the classpath :(